srcpkgs/cni-plugins/template

```diff
@@ -1,16 +1,17 @@
 # Template file for 'cni-plugins'
 pkgname=cni-plugins
-version=1.0.1
+version=1.1.1
 revision=1
 wrksrc="plugins-${version}"
 build_style=go
 go_import_path="github.com/containernetworking/plugins"
+depends="cni>=1.0.0"
```

Comment:
wait, are you sure this dependency is needed? I thought the plugins were self-contained...